question/app.js

```diff
@@ -0,0 +1,158 @@
+//基礎編
+  //Q1 変数
+  let nickname = 'ごっしー';
```

Comment:
[must]
インデントは特に不要です！
```suggestion
let nickname = 'ごっしー';
```

Reply:
承知しました。ほかの箇所のインデントも確認して修正します。